src/testsuite/arith128_test_bcd.c

```diff
@@ -41,6 +41,62 @@
 #include <testsuite/arith128_print.h>
 #include <testsuite/arith128_test_bcd.h>
 
+vui8_t
+db_vec_ZN2i128 (vui8_t zone00, vui8_t zone16)
+{
+  vui8_t result = { 0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0};
+  const vui8_t dmask = vec_splat_u8 (15);
+  const vui8_t dx10 = vec_splat_u8 (10);
+  vui8_t znd00, znd16;
+  vui8_t ones, tens;
+  vui16_t ten00, ten16;
+  vui8_t e_perm =
+#if __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__
+    { 0x00, 0x10, 0x02, 0x12, 0x04, 0x14, 0x06, 0x16, 0x08, 0x18, 0x0a, 0x1a,
+	0x0c, 0x1c, 0x0e, 0x1e };
+#else
+    { 0x01, 0x11, 0x03, 0x13, 0x05, 0x15, 0x07, 0x17, 0x09, 0x19, 0x0b, 0x1b,
+	0x0d, 0x1d, 0x0f, 0x1f };
+#endif
+  vui8_t /*i,*/ j, k, l, m/**/;
+
+  print_vint8x ("32xZoned   ", zone00);
+  print_vint8x ("           ", zone16);
+  print_vint8c ("32xZoned %c", zone00);
```

Comment:
That `%c` seems wrong.

Reply:
This is not a format string and will be printed as is. I'll change it to be less confusing.